data/presentations.yaml

```
@@ -393,4 +393,129 @@
+- title: "Securing Kafka with SPIFFE"
+  link: https://youtu.be/4pfY0uFW7yk
+  youtube: 4pfY0uFW7yk
+  presenters: Jonathan Oddy and Levani Kokhreidze
+  organizations: TransferWise
+  category: case-study
+  date: 17-Nov-2020
+  venue: SPIFFE + SPIRE at KubeCon + CloudNativeCon NA 2020
+  description: Jonathan and Levani explain how SPIFFE and Envoy is used to secure Kafka's client-broker communication, by solving common problems as certificate distribution at large scale.
+- title: "Using SPIRE in Production at Uber"
+  link: https://youtu.be/o98SFmW4akQ
+  youtube: o98SFmW4akQ
+  presenters: Andrew Moore
+  organizations: Uber
+  category: case-study
+  date: 17-Nov-2020
+  venue: SPIFFE + SPIRE at KubeCon + CloudNativeCon NA 2020
+  description: This session provides an overview of how Uber uses SPIFFE and SPIRE for workload authentication in a diverse deployment environment.
+- title: "SPIFFE at GitHub"
+  link: https://youtu.be/vX8SS5wQuY8
+  youtube: vX8SS5wQuY8
+  presenters: Eric Lee
+  organizations: GitHub
+  category: case-study
+  date: 17-Nov-2020
+  venue: SPIFFE + SPIRE at KubeCon + CloudNativeCon NA 2020
+  description: Eric explains how SPIRE is used in GitHub giving implementation details about agent operation and custom node selectors, making use of the platform-agnostic characteristic of SPIRE.
+- title: "Passport App: The role of SPIFFE and SPIRE in a return to work solution"
+  link: https://youtu.be/nZXCbiS6wfE
+  youtube: nZXCbiS6wfE
+  presenters: Frederick Kautz
+  organizations: doc.ai
+  category: case-study
+  date: 17-Nov-2020
+  venue: SPIFFE + SPIRE at KubeCon + CloudNativeCon NA 2020
+  descripion: Frederick demonstrates a SPIFFE/SPIRE enabled solution which will help employers manage there return to work strategy, focusing on privacy.
+- title: "10 Lessons From Migrating to SPIFFE After 10 Years Of Service Identity at Square"
+  link: https://youtu.be/x642wq7lbpY
+  youtube: x642wq7lbpY
+  presenters: Mat Byczkowski
+  organizations: Square
+  category: case-study
+  date: 17-Nov-2020
+  venue: SPIFFE + SPIRE at KubeCon + CloudNativeCon NA 2020
+  description: Mat exposes the principal benefits of migrating from a homegrown service identity issuer to SPIFFE, showing what the team learned in this process.
+- title: "Using a CRD to better integrate SPIRE and Kubernetes"
+  link: https://youtu.be/_6yKKvYQcak
+  youtube: _6yKKvYQcak
+  presenters: Faisal Memon
+  organizations: F5/NGINX
+  category: advanced-tutorial
+  date: 17-Nov-2020
+  venue: SPIFFE + SPIRE at KubeCon + CloudNativeCon NA 2020
+  description: In this talk the CRD for a SPIFFE ID is presented, along with its leading motivations in the context of using SPIRE at the NGINX Service Mesh Architecture.
+- title: "Fortifying Microservice Security with SPIRE and OPA"
+  link: https://youtu.be/iQ5ctLQswUc
+  youtube: iQ5ctLQswUc
+  presenters: Ash Nakar
+  organizations: OPA
```

Ash works at Styra, even though he contributes to OPA (kind of like HPE/Argeniss and SPIRE).

```suggestion
  organizations: Styra
```